build/media-suite_compile.sh

```diff
@@ -1823,6 +1825,24 @@ if [[ $bits = 64bit && $uvg266 = y ]] &&
     do_checkIfExist
 fi
 
+_check=(bin-video/vvenc/vvenc{app,FFapp}.exe)
+if [[ $bits = 64bit && $vvenc = y ]] &&
+    do_vcs "https://github.com/fraunhoferhhi/vvenc.git" vvenc; then
```

Comment:
```suggestion
    do_vcs "https://github.com/fraunhoferhhi/vvenc.git"; then
```
the third argument is unnecessary as git can infer `vvenc.git` goes to `vvenc`, it's mainly for repos that have weird names and such